app/src/main/java/ch/epfl/sweng/studyup/questions/DisplayQuestionActivity.java

```diff
@@ -186,11 +180,11 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
     private void handleTimedQuestion(Question displayQuestion) {
         String questionId = displayQuestion.getQuestionId();
         TextView time_left = findViewById(R.id.time_left); ImageView alarm = findViewById(R.id.alarmImage);
-        long now = System.currentTimeMillis(), clickedInstant = player.getClickedInstants().get(questionId);
+        long now = System.currentTimeMillis(), clickedInstant = Player.get().getClickedInstants().get(questionId);
```

Comment:
We don't check if the map actually contains the key. Could you copy-paste the code that is on the debugParty branch ?

Reply:
Yes, I'll do it